scripts/vulkaninfo_generator.py

```diff
@@ -98,12 +93,17 @@
 struct_comparisons_to_gen = ['VkSurfaceFormatKHR', 'VkSurfaceFormat2KHR', 'VkSurfaceCapabilitiesKHR',
                              'VkSurfaceCapabilities2KHR', 'VkSurfaceCapabilities2EXT']
 # don't generate these structures
-struct_blacklist = ['VkConformanceVersion']
+struct_blacklist = ['VkVideoProfilesKHR']
```

Comment:
The generator script also disallow-lists the various DRM format modifier list structures for the same reason as `VkVideoProfilesKHR`, i.e. because they contain pointers to arrays which shouldn't be used in output structures. In order to stay consistent with the Vulkan-Profiles toolchain, you probably should do the same here so that you don't accidentally generate profile JSONs containing those disallow-listed structures, as they won't comply with the schema or work with the library generator.

Reply:
The struct blacklist now contains the following.
```python
struct_blacklist = ['VkVideoProfilesKHR', 'VkVideoProfileKHR', 'VkDrmFormatModifierPropertiesListEXT', 'VkDrmFormatModifierPropertiesEXT', 'VkDrmFormatModifierPropertiesList2EXT']
```